lib/new_relic/agent/llm/embedding.rb

```diff
@@ -38,6 +39,29 @@ def attribute_name_exceptions
         def event_name
           EVENT_NAME
         end
+
+        def error_attributes(exception)
+          attrs = {}
+
+          if exception.respond_to?(:response)
```

Comment:
Similar "that's a big `if` block" comment as the previous one.

Reply:
Great suggestion! Resolved in: d91ad510ea952c64c5ff50e3f2470e433245ff9e